proto/types.proto

```diff
@@ -698,7 +698,7 @@ enum InterpreterStateType {
     INTERP_RUNNING     = 2;
     INTERP_SYNC_WAIT   = 3;
     INTERP_PAUSED      = 4;
-    INTERP_QEUEUE_WAIT = 5;
+    INTERP_QUEUE_WAIT = 5;
```

Comment:
nice find

can you please intend the equals sign to be aligned with the others so I can merge this PR